website/docs/r/express_route_circuit.html.markdown

```diff
@@ -69,6 +69,8 @@ The following arguments are supported:
 
 * `authorization_key` - (Optional) The authorization key. This can be used to set up an ExpressRoute Circuit with an ExpressRoute Port from another subscription.
 
+~> **Note:** `authorization_key` can only be set after the Circuit is created.
```

Comment:
this would indicate a logic bug in the provider, can we instead update the Create method to call the Update method so that this isn't needed?

Reply:
Got it. I will update it, call the update after the creation should cover this issue